tests/Behat/Resources/suites/ui/product/viewing_products.yaml

```diff
@@ -22,7 +25,9 @@ default:
                 - Tests\Sylius\PriceHistoryPlugin\Behat\Context\Setup\ChannelContext
                 - Tests\Sylius\PriceHistoryPlugin\Behat\Context\Setup\ProductContext
 
+                - sylius.behat.context.ui.admin.product_showpage
```

Comment:
IMO these new scenarios should be in a different suite, as this one is intended for shop

Reply:
We decided not to touch it as it is mixed in Open Source